lib/Utils.groovy

```diff
@@ -37,4 +37,14 @@ class Utils {
                 "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"
         }
     }
+
+    //
+    // Get basename of file after removing optional extension
+    //
+    public static String getFileBasename(file_string, extension='') {
```

Comment:
This feels like overkill. Why this method?

Reply:
Gone! https://github.com/nf-core/rnaseq/pull/810/commits/5169c0537b24383c3bf68d575e409031726665a8